modules/local/fgbio/filterconsensusreads/main.nf

```diff
@@ -2,14 +2,14 @@ process FGBIO_FILTERCONSENSUSREADS {
     tag "$meta.id"
     label 'process_medium_mem'
```

Comment:
"I just work here"... https://nf-co.re/docs/guidelines/components/modules#resource-requirements

>An appropriate resource label MUST be provided for the module as listed in the [nf-core pipeline template](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config#L29-L46) e.g. process_single, process_low, process_medium or process_high.

But then this:

https://nf-co.re/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines#default-processes-resource-requirements

>Sensible defaults for process resource requirements (CPUs / memory / time) for a process should be defined in conf/base.config. These should generally be specified generic with withLabel: selectors so they can be shared across multiple processes/steps of the pipeline. An nf-core standard set of labels that should be followed where possible can be seen in the [nf-core pipeline template](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config), which has the default process as a single core-process, and then different levels of multi-core configurations for increasingly large memory requirements defined with standardised labels.

Leaving it for now.

Reply:
Yes, so keep process_medium but override the memory
All the institute profiles defined in https://github.com/nf-core/configs will have the default labels set, but not process_medium_mem. This means that if e.g. a queue has to be set for their system, then it might not be.

Made a separate PR: https://github.com/nf-core/fastquorum/pull/44